js/components/uswds_checkbox_widget.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+const USWDSCheckboxWidget = function(props) {
+  return (
+    <div>
+      <input 
+        type="checkbox" 
+        id={props.label} 
+        name={props.label} 
+        className={props.value ? "checked" : "unchecked"} 
+        onClick={() => props.onChange(!props.value)} 
```

Comment:
Don't we want `onChange` instead of `onClick`?

Reply:
I'm not sure; I was following an old piece of code I had. `onChange` does seem to be more widely applicable, though.